linchpin/provision/roles/azure/tasks/provision_azure_vm.yml

```diff
@@ -99,4 +99,14 @@
   include: provisionVM.yml nameOfvm={{nameOfvm}}
   loop: "{{ name_list}}"
   loop_control:
-    loop_var: nameOfvm
\ No newline at end of file
+    loop_var: nameOfvm
```

Comment:
You don't need to add resource type here,  this is just a wrapper of VM provision, the real provision part is in provisionVM.yml